tensorboard/plugins/debugger_v2/debug_data_provider.py

```diff
@@ -95,11 +148,30 @@ def list_blob_sequences(
     def read_blob_sequences(
         self, experiment_id, plugin_name, downsample=None, run_tag_filter=None
     ):
-        del experiment_id, plugin_name, downsample, run_tag_filter  # Unused.
-        # TODO(cais): Implement this.
-        raise NotImplementedError()
+        del experiment_id, downsample  # Unused.
+        if plugin_name != PLUGIN_NAME:
+            raise ValueError("Unsupported plugin_name: %s" % plugin_name)
+        if not run_tag_filter.runs:
```

Comment:
Should be `if run_tag_filter.runs is None`, lest this also fail when
`runs` is the empty list.